src/meta/protos/proto/tenant.proto

```diff
@@ -28,3 +28,23 @@ message TIdent {
   // Resource name
   string name = 2;
 }
+
+message TQuota {
```

Comment:
`TQuota` doesn't looks good to me, how about using `TenantQuota` for better reading?